dolphinscheduler-remote/src/main/java/org/apache/dolphinscheduler/remote/utils/JacksonSerializer.java

```diff
@@ -54,7 +75,15 @@
 	 * @return deserialize type
 	 */
 	public static <T> T deserialize(byte[] src, Class<T> clazz) {
-		return JSON.parseObject(src, clazz);
+
+        String json = new String(src, StandardCharsets.UTF_8);
```

Comment:
In original logic ,if obj can't format to json ,exceptions throw. However current method return empty string ,which maybe change the logic of the invoker